clinica/pydra/machine_learning_spatial_svm/pipeline.py

```diff
@@ -0,0 +1,122 @@
+import pydra
+from pydra import Workflow
+
+from clinica.pydra.engine import clinica_io
+
+
+@clinica_io
+def build_core_workflow(name: str = "core", parameters={}) -> Workflow:
+    """Build the core workflow for the machine learning spatial svm pipeline.
+
+    Parameters
+    ----------
+    name : str, optional
+        The name of the workflow. Default="core".
+
+    parameters : dict, optional
+        Dictionary of parameters to be used
+        within the workflow.
+        Default={}.
+
+    Returns
+    -------
+    wf : Workflow
+        The core
+    """
+    from typing import Any
+
+    import clinica.pydra.machine_learning_spatial_svm.tasks as utils
+
+    # query = _build_query(parameters)
```

Comment:
I think the idea of using `_build_query` was good. It will make this function a bit easier to read.

I think we can implement it in the same way as:

https://github.com/aramis-lab/clinica/blob/a08d1c830c539ab9ec04a225a3907a8f5da1d044/clinica/pydra/statistics_volume/pipeline.py#L46-L69